compiler/utils.go

```diff
@@ -769,6 +778,17 @@ func (st signatureTypes) Param(i int, ellipsis bool) types.Type {
 	return st.VariadicType().(*types.Slice).Elem()
 }
 
+// HasResults returns true if the function signature returns something.
+func (st signatureTypes) HasResults() bool {
+	return st.Sig.Results().Len() > 0
+}
+
+// HasNamesResults returns true if the function signature returns something and
```

Comment:
Would `HasNamedResults` make more sense?

```suggestion
// HasNamesResults returns true if the function signature returns something and
```

Reply:
Yes. Strange that the linter didn't call out mismatch with the actual method name.